build/scripts/entrypoint-volume.sh

```diff
@@ -23,7 +23,8 @@ fi
 ls -la /checode/
 
 # Start the machine-exec component in background
-nohup /checode/bin/machine-exec --url '0.0.0.0:3333' &
+export MACHINE_EXEC_PORT=3333
+nohup /checode/bin/machine-exec --url "0.0.0.0:${MACHINE_EXEC_PORT}" &
```

Comment:
Che-Code will start sending the user the activity.
So, now it makes sense to enable the automatic stop of the inactive workspaces:
`--idle-timeout ${DEVWORKSPACE_IDLE_TIMEOUT}`
https://github.com/eclipse-che/che-machine-exec/blob/302ec2e92350bf77c81d4d66a8be0e0a843d1f93/cfg/cfg.go#L85
WDYT?